backend/src/main/scala/cromwell/backend/BackendJobExecutionActor.scala

```diff
@@ -56,28 +54,15 @@ trait BackendJobExecutionActor extends BackendJobLifecycleActor with ActorLoggin
   /**
     * Restart or resume a previously-started job.
     */
-  def recover: Future[BackendJobExecutionResponse]
+  def recover: Future[BackendJobExecutionResponse] = execute
```

Comment:
Oops, hit the button too quickly. I do think this would be better the way HtCondor does it though (assuming it's not a PITA to do so) where it's logging that it doesn't have recover functionality and is executing instead